templates/adversaries.html

```diff
@@ -966,25 +991,33 @@ <h3>Create a profile</h3>
                 const types = {};
                 let factsCollected = [];
                 let factsRequired = [];
+                this.undefinedAbilities = [];
 
-                this.selectedProfileAbilities.forEach((ability, index) => {
-                    let requireTypes = [];
+                this.selectedProfileAbilities.forEach((ability, index) => {	
+					let requireTypes = [];
```

Comment:
Super nit-picky, but the indentation on this line and on line 1019 are a bit off.

Reply:
Huh, it looked fine for me in VSCode, weird...